packages/swapper/src/swappers/RelaySwapper/utils/relayTokenToAssetId.ts

```diff
@@ -30,19 +31,36 @@ export const relayTokenToAssetId = (relayToken: RelayToken): AssetId => {
       return relayToken.address === RELAY_BTC_TOKEN_ADDRESS
     }
 
+    if (chainId === solanaChainId) {
+      return relayToken.address === RELAY_SOLANA_TOKEN_ADDRESS
+    }
+
     return false
   })()
 
   const { assetReference, assetNamespace } = (() => {
-    // @TODO: Handle the same for Solana and BTC
-    if (!isNativeAsset)
+    if (!isNativeAsset) {
+      const assetNamespace = (() => {
+        switch (true) {
+          case CHAIN_REFERENCE.BnbSmartChainMainnet === chainReference:
+            return ASSET_NAMESPACE.bep20
+          case isEvmChainId(chainId):
+            return ASSET_NAMESPACE.erc20
+          case CHAIN_REFERENCE.BitcoinMainnet === chainReference:
```

Comment:
Not really a thing because BTC is native only

Reply:
Removed it in [da24f87](https://github.com/shapeshift/web/pull/9253/commits/da24f87f9ab751c8b45c87a3b50e173b4cf2735b), should be useless yep!